Rakefile

```diff
@@ -1,6 +1,12 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
+begin
+  require "pact/tasks"
+rescue LoadError
+  # Pact isn't available in all environments
+end
+
```

Comment:
Actually I'm wondering if we really want this before `require File.expand_path("config/application", __dir__)`. It feels like that should be the first thing we load, because I think that sets up things with Bundler.

Reply:
Seems fair 👍